src/ethereum/ethEngine.ts

```diff
@@ -352,7 +352,7 @@ export class EthereumEngine extends CurrencyEngine<
               'call_request',
               // @ts-expect-error
               (error: Error, payload: EvmWcRpcPayload) => {
-                try {
+                const iif = async (): Promise<void> => {
```

Comment:
The try / catch block should catch all errors, both sync and async. I don't see how the IIF wrapper helps anything. What am I missing?

Reply:
The function is there to use the async/await syntax. The callback type wont allow the return of a Promise. I can use `then/catch` instead?